app/src/main/java/ar/com/wolox/android/example/ui/viewpager/random/RandomPresenter.kt

```diff
@@ -0,0 +1,44 @@
+package ar.com.wolox.android.example.ui.viewpager.random
+
+import android.util.Log
+import ar.com.wolox.android.example.model.ExampleModel
+import ar.com.wolox.android.example.utils.UserSession
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import ar.com.wolox.wolmo.core.util.Logger
+import ar.com.wolox.wolmo.core.util.ToastFactory
+import java.util.*
+import javax.inject.Inject
+
+class RandomPresenter @Inject constructor(
+        private val userSession: UserSession,
+        private val toastFactory: ToastFactory
+) : BasePresenter<IRandomView>() {
```

Comment:
Does this indentation follow Kotlin style guide?

Reply:
Yep, is the one they [recommend](https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting).